### PR TITLE
Susi logo is not correctly aligned .

### LINKED ISSUE #1551: Susi logo in footer section is not properly aligned
**Actual Behaviour**

Susi logo in footer section (https://chat.susi.ai/overview) is not properly aligned .

**Expected Behaviour**

Susi logo in footer section should be properly aligned .

**Screenshots of the issue**

logo

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #1551

Changes: Susi logo in footer (https://chat.susi.ai/overview) of 'About' section is aligned properly .

Demo Link: https://pr-1552-fossasia-susi-web-chat.surge.sh

Screenshots for the change: 
logo

### CODE REVIEW

**Comment by samagragupta** [comment]
@tstreamDOTh Done .

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
+@media only screen and (max-width : 800px) {
```
Comment (anshumanv) [inline src/components/Footer/Footer.css]: Shift media queries at the end of the file

**Comment by samagragupta** [comment]
@anshumanv The travis CI didn't deploy the build.
error

**Comment by samagragupta** [comment]
@anshumanv please review the changes .

**Comment by anshumanv** [comment]
Looks good, try to squash your commits.

**Pull request closed without merging**